src/loaders/genome_attributes/compute_genome_attribs.py

```diff
@@ -185,6 +185,41 @@ def _run_gtdb_tk_classify_wf(batch_file_path, work_dir, debug, genome_ids, progr
         f'{len(genome_ids)} genomes')
 
 
+def _map_tool_id_to_genome_id(tool_name, original_genome_id, genome_file_name):
```

Comment:
I thought we were going to save the path to the input in this file.

Reply:
yea. We were talking about doing `genome_id`: `source_genome_file`. But when I implemented it, I thought again it made no sense to do that because `genome_id`: `source_genome_file` information is already embedded into the file structure. But more importantly, the parser cannot use `genome_id`: `source_genome_file` information directly to identify the genome identifier for each tool. The parser will again parse the `source_genome_file` based on different tools, as it is currently doing.

Reply:
is there a reason not to have both the file and the genome id?

Reply:
`both the file and genome id`? The metadata file created has `tool_genome_identifer: genome_id`. The source file and genome id mapping are embedded already into the file structure. The folder name is the `genome_id` and the files inside are the source files downloaded from NCBI. Does that answer your question? I might misunderstand your question.

Reply:
I understand you're saying it's redundant - what I'm asking is if there's a reason not to put this redundant information into the csv file just to make it easy for someone reading the file to figure out where the source file is

Reply:
Why add another file rather than another column in the current TSV file?

Reply:
👍 added source genome file column.

Reply:
:+1: